utils/tests_fetcher.py

```diff
@@ -367,6 +367,34 @@ def get_diff_for_doctesting(repo: Repo, base_commit: str, commits: List[str]) ->
     return code_diff
 
 
+def get_all_doctest_files() -> List[str]:
+    """
+    Return the complete list of python and Markdown files on which we run doctest.
+
+    At this moment, we restrict the files to in `src/` or `docs/source/en/` that are not in `utils/not_doctested.txt`.
+
+    Returns:
+        `List[str]`: The complete list of Python and Markdown files on which we run doctest.
```

Comment:
```suggestion
        `List[str]`: The filtered list of Python and Markdown files on which we run doctest.
```

Reply:
If we add `filtered`, it's also confusing that what `get_doctest_files` means - that is used for CircleCI and have one more filtration